guides/integrate-wordpress.md

```diff
@@ -128,11 +127,11 @@ Congratulations, you have configured the BI console and the OIDC client plugin.
 
```

Comment:
What do you think about mentioning bi-sdk-js to learn about binding credentials and such that that developer will need to do along with this provider?